transport/transport-runtime/src/main/java/com/google/android/datatransport/runtime/scheduling/jobscheduling/JobInfoScheduler.java

```diff
@@ -81,7 +81,12 @@ public void schedule(TransportContext transportContext, int attemptNumber) {
     // Check if there exists a job scheduled for this backend name.
     if (isJobServiceOn(jobScheduler, jobId)) return;
     // Obtain the next available call time for the backend.
-    long timeDiff = eventStore.getNextCallTime(transportContext) - clock.getTime();
+    Long backendTime = eventStore.getNextCallTime(transportContext);
+
+    long timeDiff = 0;
+    if (backendTime != null) {
+      timeDiff = backendTime - clock.getTime();
+    }
     // Schedule the build.
     PersistableBundle bundle = new PersistableBundle();
     bundle.putInt(SchedulerUtil.ATTEMPT_NUMBER, attemptNumber);
```

Comment:
`setMinimumLatency()` below is wrong, you should not add offset to `clock.now()`.